plugins/uf2/fu-uf2-device.c

```diff
@@ -408,6 +408,7 @@ fu_uf2_device_init(FuUf2Device *self)
 	fu_device_add_protocol(FU_DEVICE(self), "com.microsoft.uf2");
 	fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_UNSIGNED_PAYLOAD);
 	fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_CAN_VERIFY_IMAGE);
+	fu_device_add_internal_flag(FU_DEVICE(self), FU_DEVICE_INTERNAL_FLAG_REPLUG_MATCH_GUID);
```

Comment:
do you think that should match *all* UF2 devices, or just your device?

Reply:
What is the best way to do this per-device?

Reply:
I believe that, similar to DFU, it will happen often that UF2 is exposed only in a bootloader mode that must be triggered, then after flashing, that mode will disappear. I'm not sure if this flag is the only change required to support this.

Reply:
Yup, I think that's probably true -- lets leave the FU_DEVICE_INTERNAL_FLAG_REPLUG_MATCH_GUID for all devices.